client/src/components/Public/EntitySearch/EntitySearch.js

```diff
@@ -14,19 +18,22 @@ type EntitySearchProps = {
   toggleEntitySearchOpen: () => Object,
   entitySearchEids: Array<number>,
   entitySearchFor: string,
+  entitySearchLoaded: boolean,
 }
 
 const EntitySearch = ({
   toggleEntitySearchOpen,
   entitySearchEids,
   entitySearchFor,
+  entitySearchLoaded,
 }: EntitySearchProps) => (
   <div className="search-results">
     <div className="search-results-header">
       <button type="button" className="close" onClick={toggleEntitySearchOpen}>
         <span>&times;</span>
       </button>
-      {entitySearchFor && `${resultPlurality(entitySearchEids.length)} pre "${entitySearchFor}".`}
+      {entitySearchLoaded &&
```

Comment:
Please, hide the whole `div`, because now, while loading, it looks really strange. There is also some scrollbar which shouldn't be there, check that, too.